forum/qa-plugin/report-reason/script.js

```diff
@@ -45,51 +34,40 @@ $(document).ready(function()
                  cache: false,
                  success: function(data)
                  {
-                    if(typeof data.error !== "undefined")
+                    if(data.error)
                     {
-                        console.log(data.error);
+            
+                        errorText = 'Blad serwera. Prosze sprobowac za jakis czas';
+                        console.log(data);
                         alert('Blad serwera. Prosze sprobowac za jakis czas');
                     }
-                    else if(typeof data.success !== "undefined")
+                    else if(data.success)
                     {
                         // if success, reload page
                         location.reload();
                     }
                     else
                     {
-                        console.log(data);
+                        errorText = 'Blad serwera. Prosze sprobowac za jakis czas';
```

Comment:
Jeśli dobrze widzę, to w linijce 56 jest ten sam tekst. Przypisz go zatem do zmiennej i ją podstawiaj.